io/zenoh-transport/src/unicast/universal/link.rs

```diff
@@ -38,58 +38,66 @@ use zenoh_sync::{RecyclingObject, RecyclingObjectPool, Signal};
 #[derive(Clone)]
 pub(super) struct TransportLinkUnicastUniversal {
     // The underlying link
-    pub(super) link: TransportLinkUnicast,
+    pub(super) link: Arc<TransportLinkUnicast>,
     // The transmission pipeline
     pub(super) pipeline: Option<TransmissionPipelineProducer>,
     // The transport this link is associated to
     transport: TransportUnicastUniversal,
     // The signals to stop TX/RX tasks
-    handle_tx: Option<Arc<async_executor::Task<()>>>,
+    handle_tx: Option<Arc<JoinHandle<()>>>, //Option<Arc<async_executor::Task<()>>>,
```

Comment:
You need to use the `TransportExecutor` to spawn `tx` tasks, not the `async_std` global one.